dataactvalidator/config/sqlrules/d32_detached_award_financial_assistance_1.sql

```diff
@@ -0,0 +1,17 @@
+-- PeriodOfPerformanceStartDate is an optional field, but when provided, must follow YYYYMMDD format
+CREATE OR REPLACE function pg_temp.is_date(str text) returns boolean AS $$
+BEGIN
+  perform CAST(str AS DATE);
+  return TRUE;
+EXCEPTION WHEN others THEN
+  return FALSE;
+END;
+$$ LANGUAGE plpgsql;
+
+SELECT
+    row_number,
+    period_of_performance_star
+FROM detached_award_financial_assistance
+WHERE submission_id = {0}
+    AND period_of_performance_star IS NOT NULL
```

Comment:
do we want this to fail if it's blank? or should we treat blank and NULL the same?